app/src/main/java/com/yml/womensafety/navigationdrawer/home/HomePageActivity.kt

```diff
@@ -54,6 +54,32 @@ class HomePageActivity : AppCompatActivity() {
         } else {
             requestPermission()
         }
+
+        val intentService = Intent(this@HomePageActivity, AccelerometerService::class.java)
+        startService(intentService)
+
+        val myReceiver: BroadcastReceiver = object : BroadcastReceiver() {
+            override fun onReceive(context: Context, intent: Intent?) {
+                if ("com.yml.womensafety.ACTION" == intent?.action) {
+                    val receivedText = intent.getStringExtra("com.yml.womensafety.EXTRA_TEXT")
+
+                    sendSms()
+                    val dialogBuilder = AlertDialog.Builder(context)
```

Comment:
Use the method in util class created by @Annapoornakamath98 To show dialog

Reply:
Done.